### PR TITLE
docs: add note that CLI version 15 doesn't create karma.conf.js in new projects

### LINKED ISSUE #48364: Update docs on how to create code coverage in the current Angular version (15)
### Which @angular/* package(s) are the source of the bug?

Don't known / other

### Is this a regression?

No

### Description

content of this file is outdated. newly generatec angular 15 projects do no contain karma.conf.js

https://angular.io/guide/testing-code-coverage

### Please provide a link to a minimal reproduction of the bug

karma.conf.js

### Please provide the exception or error you saw

```true
no karma.conf.js present
```

### Please provide the environment you discovered this bug in (run `ng version`)

```true
Angular CLI 15
```

### Anything else?

no

### PR DESCRIPTION
**Pull request by StephenYu2018**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

*aio/content/guide/testing-code-coverage.md* doesn't mention that Angular CLI version 15 doesn't automatically create `karma.conf.js` file in new projects. The issue can be found [here](https://github.com/angular/angular/issues/48364).

Closes : #48364

## What is the new behavior?
*testing-code-coverage.md* now displays an informational alert summarizing the update and provides a link to *aio/content/guide/testing.md#configuration* for more detailed instructions to manually setup `karma.conf.js`. Certain words were also simplified based on the linter error messages from Vale, and CLI now explicitly refers to the Angular CLI.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
N/A

### CODE REVIEW

**Review by hexiaokang — APPROVED** [review]

File: aio/content/guide/testing-code-coverage.md [hunk 1]
```diff
@@ -2,27 +2,27 @@
...
 # Find out how much code you're testing
 
-The CLI can run unit tests and create code coverage reports.
+The Angular CLI can run unit tests and create code coverage reports.
 Code coverage reports show you any parts of your code base that might not be properly tested by your unit tests.
 
 <div class="alert is-helpful">
 
-If you'd like to experiment with the application that this guide describes, <live-example name="testing" noDownload>run it in your browser</live-example> or <live-example name="testing" downloadOnly>download and run it locally</live-example>.
+If you'd like to experiment with the app that this guide describes, <live-example name="testing" noDownload>run it in your browser</live-example> or <live-example name="testing" downloadOnly>download and run it locally</live-example>.
```
Comment (alan-agius4) [inline aio/content/guide/testing-code-coverage.md:10]: Why this change? Application should be preferred over app.

File: aio/content/guide/testing-code-coverage.md [hunk 2]
```diff
@@ -37,11 +37,19 @@ If you want to create code-coverage reports every time you test, set the followi
 ## Code coverage enforcement
 
 The code coverage percentages let you estimate how much of your code is tested.
-If your team decides on a set minimum amount to be unit tested, enforce this minimum with the Angular CLI.
+If your team decides on a set smallest amount to be unit tested, enforce this amount with the Angular CLI.
```
[the remark this answers is not in the document]
Comment (StephenYu2018) [reply aio/content/guide/testing-code-coverage.md]: Sorry, I only saw this comment now, and I ended up using "least". Should I also change all the instances where "minimum" was used back to its original state, or is it just the instance I ended up using "smallest"?

File: aio/content/guide/testing-code-coverage.md [hunk 3]
```diff
@@ -37,11 +37,19 @@ If you want to create code-coverage reports every time you test, set the followi
 ## Code coverage enforcement
 
 The code coverage percentages let you estimate how much of your code is tested.
-If your team decides on a set minimum amount to be unit tested, enforce this minimum with the Angular CLI.
+If your team decides on a set least amount to be unit tested, enforce this amount with the Angular CLI.
```
Comment (alan-agius4) [inline aio/content/guide/testing-code-coverage.md:40]:
```suggestion
If your team decides on a set a minimum amount to be unit tested, enforce this minimum with the Angular CLI.
```

File: aio/content/guide/testing-code-coverage.md [hunk 4]
```diff
@@ -37,11 +37,19 @@ If you want to create code-coverage reports every time you test, set the followi
...
 
-For example, suppose you want the code base to have a minimum of 80% code coverage.
+For example, suppose you want the code base to have at least 80% code coverage.
```
Comment (alan-agius4) [inline aio/content/guide/testing-code-coverage.md:42]:
```suggestion
For example, suppose you want the code base to have a minimum of 80% code coverage.
```

File: aio/content/guide/testing-code-coverage.md [hunk 5]
```diff
@@ -63,7 +71,7 @@ coverageReporter: {
 
 </code-example>
 
-The `check` property causes the tool to enforce a minimum of 80% code coverage when the unit tests are run in the project.
+The `check` property causes the tool to enforce at least 80% code coverage when the unit tests are run in the project.
```
Comment (alan-agius4) [inline aio/content/guide/testing-code-coverage.md:74]:
```suggestion
The `check` property causes the tool to enforce at a minimum of 80% code coverage when the unit tests are run in the project.
```

File: aio/content/guide/testing-code-coverage.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (alan-agius4) [reply aio/content/guide/testing-code-coverage.md]: Suggested the changes.

**Comment by alan-agius4** [comment]
@StephenYu2018, couple of nits above. Also, please squash the commits.

**Review by alan-agius4 — CHANGES_REQUESTED** [review]
@StephenYu2018, couple of nits above. Also, please squash the commits.

File: aio/content/guide/testing-code-coverage.md [hunk 6]
```diff
@@ -11,18 +11,18 @@ If you'd like to experiment with the application that this guide describes, <liv
 
 </div>
 
-To generate a coverage report run the following command in the root of your project.
+To create a coverage report run the following command in the root of your project.
```
Comment (alan-agius4) [inline aio/content/guide/testing-code-coverage.md:14]:
```suggestion
To generate a coverage report run the following command in the root of your project.
```

**Review by alan-agius4 — APPROVED** [review]

**Comment by alan-agius4** [comment]
Hi @StephenYu2018,

Thanks for your contribution but this PR has been superseded by a larger scoped one (https://github.com/angular/angular/pull/48757)

**Comment by StephenYu2018** [comment]
Thanks for the opportunity of my first documentation pull request!
